project_one/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>project one</title>
+    <link rel="stylesheet" type="text/css" href="stylesheet.css">
+</head>
+<body>
+    <a id= "grading-element-two" href ="about.html">About</a>
+    <a href = "index.html">Home</a>
+    <div id= "heading">
+        <h1>Project One</h1>
+
+    </div>
+    <br>
+    <br>
+    <button id="grading-element-one" onclick="addingText()">Your DOM will suffer consequences</button>
+
+    <div id= "grading-element-one-consequence">
+        
+        
+    
+    </div>
+    
+    <br>
+    <br>
+    <button id= "grading-element-three" onclick="showing()">Show the ninja div</button>
+    <br>
+    <div id ="grading-element-three-consequence">
+
+        <p>the div ninja was discover, is not a good ninja :(</p>
+        
+    </div>
+
+    
+   <div id ="grading-element-five">
+       <p>This text change color when you hover.</p>
+
+   </div>
+      <form action=" http://localhost:3000/form " name="grading-element-four" onsubmit="return checkedForm()" method="POST">
+    
+
+    First name:<br>
+    <input type="text" name="testOne"  value= "" >
+    <br>
+    Last name:<br>
+    <input type="text" name="testTwo" value = "">
+    <br>
+    Age:<br>
+    <input type="number" name="testThree" min="0" value="">
+    <br><br>
+    <input id="submit" type="submit" value="Submit">
+  </form>
+  
+
+<script type = "text/javascript">
+
+    function addingText() {
+        
+        var someText = document.createElement("P");
+        someText.innerText = "consequence";
+        document.getElementById("grading-element-one-consequence").appendChild(someText);

```

Comment:
Very close. Give `someText` the id `grading-element-one-consequence` via javascript and append it to an element.